.bumpversion.cfg

```
@@ -20,3 +20,5 @@ values =
 [bumpversion:file:VERSION]
 
 [bumpversion:file:setup.py]
+
+[bumpversion:file:deepbots/__init__.py]
```

```suggestion
[bumpversion:file:deepbots/__init__.py]
search = __version__ = "{current_version}"
replace = __version__ = "{new_version}"
```
Ref.: https://github.com/wandb/client/blob/93aa52d6d411b24525809176d203611d4fa07e39/setup.cfg#L34-L36